### DISCUSSION TITLE
Add to Calendar

### DISCUSSION THREAD

**Discussion by zwannie** [2Abendsegler/GClh#1962] [Questions] [answered]
At an event cache, I see that the option "Add to Calendar" is gone.
Can I have it back?

**Comment by capoaira** [comment, marked as answer]
It's a bug from GS. I think we'll wait to see if they fix it. If not, we can try to implement the function.

**Comment by 2Abendsegler** [comment]
( https://forums.geocaching.com/GC/index.php?/topic/368158-add-to-calendar-link-missing )
